src/hugr/rewrite/outline_cfg.rs

```diff
@@ -162,7 +162,8 @@ impl Rewrite for OutlineCfg {
         for (pred, br) in preds {
             if !self.blocks.contains(&pred) {
                 h.disconnect(pred, br).unwrap();
-                h.connect(pred, br, new_block, 0).unwrap();
+                h.connect(pred, br.as_outgoing().unwrap(), new_block, 0)
```

Comment:
You could avoid the conversion by using `preds = h.linked_outputs(...)` above.

Reply:
Good spot! :)